XboxKeyboardMouse/Forms/Options.cs

```diff
@@ -166,38 +147,38 @@ public enum MouseButton {
 
         private void LoadXboxInputButtons() {
             // Keyboard
-            xbo_k_A.Text = ((System.Windows.Input.Key)cfg.Controls_KB_Xbox_A).ToString();
-            xbo_k_B.Text = ((System.Windows.Input.Key)cfg.Controls_KB_Xbox_B).ToString();
-            xbo_k_X.Text = ((System.Windows.Input.Key)cfg.Controls_KB_Xbox_X).ToString();
-            xbo_k_Y.Text = ((System.Windows.Input.Key)cfg.Controls_KB_Xbox_Y).ToString();
+            xbo_k_A.Text = ((Key)cfg.Controls_KB_Xbox_A).ToString();
```

Comment:
Can we use camelcase here instead of underscores for all these vars to be consistent?

Reply:
@fqlx got it, i'll work on it when i get home.

Reply:
@CallumCarmicheal probably best to wait for the merge to avoid same-line conflicts

Reply:
@fqlx Should i do that for the Config, Form or both?

Reply:
If everything is camelcase, we should be consistent for readability. So change the underscore vars to camelcase.